styles.css

```diff
@@ -12,17 +13,45 @@ body {
     align-items: center;
     min-height: 100vh;
 }
+
+.input-box {
+    position: relative;
+}
+
 .error-text,
 .error-icon {
     display: none;
 }
+
 .error {
-	border: 2px solid hsl(0, 100%, 74%);
+	border: 3px solid;
+    border-radius: 5px;
+    color: hsl(0, 100%, 74%);
```

Comment:
What is the difference between this error class and the border input box mentioned below?

Reply:
the error class is from JS, we put a class in JS. hmmm maybe it should be input box?